### PR TITLE
Add schema for user and movie

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: horse/web/recommendations.py [hunk 1]
```diff
@@ -2,18 +2,20 @@
 from flask_restful import Resource, Api
 
 from horse.recommendation.dummy_service import DummyRecommendationService
+from horse.web.schemas.movie import MovieSchema
```
Comment (biern) [inline horse/web/recommendations.py]: I think we can just import `movies_schema` and not have to re-instantiate it here

File: horse/web/schemas/user.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    followed_users = fields.Nested(
+        'self', only=('pk', 'name'), dump_only=True, many=True
+    )
+    liked_movies = fields.Nested(MovieSchema, dump_only=True, many=True)
+
+    @pre_dump
+    def load_movies_and_users(self, obj):
+        obj.followed_users = obj.get_followed_users()
+        obj.liked_movies = obj.get_liked_movies()
+        return obj
```
Comment (biern) [inline horse/web/schemas/user.py:18]: Well that modifies the source object, that's not pretty. Is there any other way?

File: tests/integration/test_users.py [hunk 3]
```diff
@@ -39,6 +45,13 @@ def test_user_can_follow_another_user(client, users_repo):
     assert 'Adam' in followed_users_names
 
 
+def test_user_follow_another_user_400_error(client):
+    response = client.post('/users/1/follow', data=json.dumps({
+        'user_pk': '2',
+    }), content_type='application/json')
+    assert response.status_code == 400
+
+
```
Comment (biern) [inline tests/integration/test_users.py:54]: Testing every endpoint for `400` errors will become repetitive, but until we have a general solution (like a base `MarshmallowResource` handling errors) I think it's ok.

**Review by biern — CHANGES_REQUESTED** [review]
I like it, left a few questions

File: horse/web/schemas/user.py (hunk 2 above)
Comment (pgrzesik) [reply horse/web/schemas/user.py:18]: It's harder than it looks in marshmallow, at least i didn't found a simple way to do that, I'm going to remove those fields in post_dump

File: horse/web/recommendations.py (hunk 1 above)
Comment (pgrzesik) [reply horse/web/recommendations.py]: right, fixed

File: horse/web/schemas/user.py (hunk 2 above)
Comment (pgrzesik) [reply horse/web/schemas/user.py:18]: it's also not possible beacause post_dump receives serialized object, i'll create an issue to fix that

Comment (biern) [reply horse/web/schemas/user.py:18]: Roger

**Pull request merged**